invenio_communities/views/decorators.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+from functools import wraps
+from flask import g
+from invenio_communities.proxies import current_communities
+
+
+def service():
+    """Get the record service."""
+    return current_communities.communities_service
```

Comment:
```suggestion
    return current_communities.service
```

Reply:
@lnielsen I prefixed it because in the future we might have different services for members and requests, no? Or should we follow the `record_service.<subservice>` way?